xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/xwiki.js

```diff
@@ -1383,6 +1383,7 @@ if (htmlElement.readAttribute('data-xwiki-reference') != null) {
   XWiki.Document.currentDocumentReference = XWiki.currentDocument.getDocumentReference();
 }
 XWiki.Document.URLTemplate = "$xwiki.getURL('__space__.__page__', '__action__')";
+XWiki.Document.ViewURLTemplate = "$xwiki.getURL('__space__.__page__', 'view')";
```

Comment:
Please note that it is not sufficient to just cherry pick this to the 7.2 branch, as jumpToPage.js has recently been refactored.  There are separate templates for jumpToPage.js in 7.2.